specificatie/openapi.yaml

```diff
@@ -910,15 +910,16 @@ components:
 
     IntegerFilter:
       type: object
-      description: >-
-        Geeft aan met welke waarden je kunt zoeken.
-        Met `min` zoek je naar waarden groter of gelijk aan de opgegeven waarde.
-        Met `max` zoek je naar waarden kleiner dan de opgegeven waarde.
+      description: "Zoek binnen een bereik tussen de opgegeven minimumwaarde en maximumwaarde. De opgegeven minimumwaarde moet lager zijn dan de opgegeven maximumwaarde."
       properties:
         min:
           type: integer
+          minimum: 1
```

Comment:
Hier kan niet 1 worden opgegeven, want een bouwjaar mag theoretisch 0000 zijn.
De minimum waarde voor Integer moet dan 0 zijn, echter in de toekomst kunnen evt. parameters worden toegevoegd met bv. grenswaarden -60 tot +60, als je het minimum hier op 0 zet kun je dit type dan niet meer gebruiken voor die parameter gebruiken en je kunt hier ook geen -60 meer van maken omdat de validatie van andere parameters dan niet meer goed gaat. Je kunt dus beter de minimum en maximum waarde in de beschrijving bij de parameter zetten, of zoals eerder voorgesteld een apart Filter type definiëren voor bouwjaar en oppervlakte en daarin de exacte minimum en maximum waarde voor het betreffende attribuut zetten.